frontend/src/component/personalDashboard/MyProjects.tsx

```diff
@@ -149,7 +152,7 @@ export const MyProjects = forwardRef<
         const state: MyProjectsState = projects.length
             ? personalDashboardProjectDetails
                 ? 'projects'
-                : 'projects with error'
+                : 'projects with error or loading'
```

Comment:
so does it mean that on error we're stuck with the skeleton?

Reply:
Nah. If we get an error the use loading ref is false, so we're good.